### PR TITLE
fix: hardcode sports filter into keywordOrSet1 for queries

### PR DESCRIPTION
**Pull request by kristianjokela**
## Description
Hardcodes sports filters for sports queries, so they will only return results including either "liikunta" or "urheilu" keywords.
### Closes

**[LIIKUNTA-373](https://helsinkisolutionoffice.atlassian.net/browse/LIIKUNTA-373):**

## Additional notes

[LIIKUNTA-373]: https://helsinkisolutionoffice.atlassian.net/browse/LIIKUNTA-373?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by terovirtanen** [comment]
**Sports-Helsinki branch is deployed to platta: https://liikunta-pr177.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**Sports-Helsinki branch is deployed to platta: https://liikunta-pr177.dev.hel.ninja** :rocket::rocket::rocket:

**Review by karisal-anders — COMMENTED** [review]
Left a comment about the used keywords.

File: apps/sports-helsinki/src/domain/search/eventSearch/utils.tsx
Comment (kristianjokela) [reply apps/sports-helsinki/src/domain/search/eventSearch/utils.tsx:176]: OK, I'll test this with this list, let's see how much it impacts the query.

Comment (nikomakela) [reply apps/sports-helsinki/src/domain/search/eventSearch/utils.tsx:176]: I think the most important ones would be
- https://api.hel.fi/linkedevents/v1/keyword/yso:p916/ -> liikunta
- https://api.hel.fi/linkedevents/v1/keyword/yso:p965/ -> urheilu

In that list there are keywords like "basketball" and "football", e.g. https://api.hel.fi/linkedevents/v1/keyword/yso:p6409/ is "jalkapallo". I'm not sure whether we should have that keyword in any of our lists. So probably the contant should be used, but the constant content could be revalidated from the product owner.

Comment (kristianjokela) [reply apps/sports-helsinki/src/domain/search/eventSearch/utils.tsx]: It doesn't seem to slow down the query too much - at least on my local tests - so I'd say it's OK to use this list.

**Comment by terovirtanen** [comment]
**Sports-Helsinki branch is deployed to platta: https://liikunta-pr177.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://liikunta-pr177.dev.hel.ninja :laughing::tada::tada::tada:

**Review by karisal-anders — APPROVED** [review]
LGTM, approved as long as all the tests pass. One thing you could IMHO still do—but I leave this up to you to decide what to do here if anything—is to combine the 4 commits into one commit for clarity.

**Comment by terovirtanen** [comment]
**Sports-Helsinki branch is deployed to platta: https://liikunta-pr177.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://liikunta-pr177.dev.hel.ninja :laughing::tada::tada::tada:

**Comment by terovirtanen** [comment]
**Sports-Helsinki branch is deployed to platta: https://liikunta-pr177.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
**Sports-Helsinki branch is deployed to platta: https://liikunta-pr177.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __failed__ for https://liikunta-pr177.dev.hel.ninja :crying_cat_face::anger::boom::boom:

**Comment by terovirtanen** [comment]
**Sports-Helsinki branch is deployed to platta: https://liikunta-pr177.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __failed__ for https://liikunta-pr177.dev.hel.ninja :crying_cat_face::anger::boom::boom:

**Comment by terovirtanen** [comment]
## TestCafe result is __failed__ for https://liikunta-pr177.dev.hel.ninja :crying_cat_face::anger::boom::boom:

**Comment by terovirtanen** [comment]
**Sports-Helsinki branch is deployed to platta: https://liikunta-pr177.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __failed__ for https://liikunta-pr177.dev.hel.ninja :crying_cat_face::anger::boom::boom:

**Comment by terovirtanen** [comment]
**Sports-Helsinki branch is deployed to platta: https://liikunta-pr177.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __failed__ for https://liikunta-pr177.dev.hel.ninja :crying_cat_face::anger::boom::boom:

**Comment by terovirtanen** [comment]
**Sports-Helsinki branch is deployed to platta: https://liikunta-pr177.dev.hel.ninja** :rocket::rocket::rocket:

**Comment by terovirtanen** [comment]
## TestCafe result is __success__ for https://liikunta-pr177.dev.hel.ninja :laughing::tada::tada::tada:

**Pull request merged**
